src/main/java/Main.java

```diff
@@ -0,0 +1,46 @@
+import db.Storage;
+import dto.FruitTransactionDto;
+import java.util.List;
+import java.util.Map;
+import model.Fruit;
+import service.DataParser;
+import service.impl.CsvFruitDataReaderService;
+import service.impl.FruitDataParser;
+import service.operations.OperationHandler;
+import service.operations.impl.BalanceOperationHandler;
+import service.operations.impl.PurchaseOperationHandler;
+import service.operations.impl.ReturnOperationHandler;
+import service.operations.impl.SupplyOperationHandler;
+import service.operations.report.ReportGenerator;
+import service.operations.report.ReportWriter;
+import strategy.OperationStrategy;
+
+public class Main {
+    private static final String reportPath = "src/main/java/resources/report.csv";
+    private static final String inuptPath = "src/main/java/resources/input";
+
+    public static void main(String[] args) {
+        Storage storage = new Storage();
+        BalanceOperationHandler balanceOperationHandler = new BalanceOperationHandler(storage);
+        PurchaseOperationHandler purchaseOperationHandler = new PurchaseOperationHandler(storage);
+        ReturnOperationHandler returnOperationHandler = new ReturnOperationHandler(storage);
+        SupplyOperationHandler supplyOperationHandler = new SupplyOperationHandler(storage);
+        List<OperationHandler> handlers = List.of(balanceOperationHandler,
```

Comment:
let's better create Map<Operation, OperationHandler> and then extract from this map necessary handler by operation